tests/fixtures/app_store_tests/core/templates.yml

```diff
@@ -0,0 +1,195 @@
+# Copyright (c) 2013 Shotgun Software Inc.
+# 
+# CONFIDENTIAL AND PROPRIETARY
+# 
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your 
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+#
+# This file is one of the central points in the Tank configuration and a counterpart to
+# the folder configuration structure.
+#
+# the folder structure underneath the project folder is used to create folders on disk - 
+# templates.yml (this file) refers to those folders. Therefore, the two files need to be 
+# in sync. This file contains an overview of all locations that are used in Tank.
+#
+#
+# Whenever an app or an engine refers to a location on disk, it is using a entry defined in 
+# this file. For more information, see the Tank Documentation. 
+
+keys:
```

Comment:
is this file intentionally extensive? Would it make sense to cut it down to a minimum?